client/app/bundles/KudosApp/components/KudosList.jsx

```diff
@@ -69,7 +78,7 @@ export default class KudosList extends React.Component {
   render() {
     return <div className="kudos-list__container">
       <TabBarContainer />
-      <List giverId={this.props.id} kudos={this.props.kudos} likeKudo={this.props.likeKudo} unlikeKudo={this.props.unlikeKudo} updateKudo={this.props.updateKudo} />
+      <List userId={this.props.id} kudos={this.props.kudos} likeKudo={this.props.likeKudo} unlikeKudo={this.props.unlikeKudo} updateKudo={this.props.updateKudo} />
       {this.props.isFetchingKudos ? <Spinner /> : null}
```

Comment:
I noticed on prod too that the `List` will say 'No Kudos' even while the spinner is going.  Can we change these two lines to be:
```js
this.props.isFetchingKudos ? <Spinner /> : <List ... />
```

Reply:
Instead I decided to change the logic for display 'No kudos' only if its length 0 AND it's not fetching (which should indicate that a fetch has been attempted already).

I did this because the spinner will be probably be able to be re-used for scroll-loading, and I wouldn't want the entire list of currently loaded kudos to disappear while fetching additional kudos.

Reply:
^oh yeah I forgot about that, good call